x-pack/plugins/infra/public/alerting/inventory/components/expression_chart.tsx

```diff
@@ -192,7 +191,7 @@ export const ExpressionChart: React.FC<Props> = ({
             tickFormat={dateFormatter}
           />
           <Axis id={'values'} position={Position.Left} tickFormat={yAxisFormater} domain={domain} />
-          <Settings theme={getChartTheme(isDarkMode)} />
+          <Settings baseTheme={chartTheme.baseTheme} />
```

Comment:
Hey @maryam-saeidi , previously `getChartTheme` was returning either `DARK_THEME` or `LIGHT_THEME` from `@elastic/charts` and was passing it to `theme`, but they are now actually `baseTheme`.

Looking at the `useChartBaseTheme`, it returns the same `DARK_THEME` and `LIGHT_THEME` (see [here](https://github.com/elastic/kibana/blob/main/src/plugins/charts/public/services/theme/theme.ts#L96.)). 

Passing `theme` to those charts that don't have it would change the chart styling, adding these horizontal lines. Nothing bad, but I didn't want to change it in this PR, as it's not its focus

with:
<img width="572" alt="image" src="https://github.com/elastic/kibana/assets/2767137/b29ccdc0-1300-46b5-a127-5cff49e1b4f2">

without

<img width="572" alt="image" src="https://github.com/elastic/kibana/assets/2767137/11e73f04-3368-45d2-954e-dd0c52d07808">

Seems like something to be addressed in a separate PR. wdyt?

Reply:
I see, thanks for the clarification, make sense :)